infra/examples/aws-msft-365/main.tf

```diff
@@ -12,9 +12,22 @@ terraform {
     }
   }
 
-  # if you leave this as local, you should backup/commit your TF state files
+  # we recommend you use a secure location for your Terraform state (such as S3 bucket), as it
+  # may contain sensitive values (such as API keys) depending on which data sources you configure.
+  #
+  # local may be safe for production-use IFF you are executing Terraform from a secure location
```

Comment:
```suggestion
  # local may be safe for production-use IF you are executing Terraform from a secure location
```

Reply:
sry, `IFF` is math notation for "if and only if"